COMMUNICATON.md

```diff
@@ -0,0 +1,33 @@
+# Communication
+
+## Code of conduct
+
+This project has adopted the [GitHub community guidelines](https://help.github.com/en/github/site-policy/github-community-guidelines).
```

Comment:
Happy to see this here. There's also [PSF's CoC](https://www.python.org/psf/conduct/) which has nice procedures to handle incidents and is quite adopted by the python community. I don't mind either one.

Reply:
I stole this line from our CONTRIBUTING.md
When I had this as a separate file I wanted to make sure the CoC shows up. I'll follow Miro's suggestion to not have a separate file, so I probably won't need it anymore since we have it in CONTRIBUTING.md
If you have a strong preference towards PSF's CoC let me know and we can have a broader discussion about it! It may not be part of this PR then, though.

Reply:
All good. It's not a bad CoC anyway.